flight-service/src/main/java/com/ulrich/matthiae/spring/clouddemo/flight/FlightServiceApplication.java

```diff
@@ -4,12 +4,14 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker;
 import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
+import org.springframework.cloud.netflix.hystrix.EnableHystrix;
 import org.springframework.cloud.openfeign.EnableFeignClients;
 
 @SpringBootApplication
-@EnableFeignClients("com.ulrich.matthiae.spring.clouddemo.flight")
+@EnableFeignClients
 @EnableDiscoveryClient
 @EnableCircuitBreaker
+@EnableHystrix
```

Comment:
Do you need this as well when @EnableCircuitBreaker is specified?

Reply:
nah - but you need EnableHystrix for actuator monitor endpoint for turbine - will clean up